internal/config/config.go

```diff
@@ -10,8 +10,8 @@ import (
 const (
 	DefaultFileName = "nri-kubernetes"
 	DefaultFilePath = "/etc/newrelic-infra"
-	DefaultTimeout  = 10000 * time.Millisecond
-	DefaultRetries  = 4
+	DefaultTimeout  = 3 * time.Second
```

Comment:
I would keep it 10 seconds. The charts use that for contacting kubelet and ksm and I do not see a drawback in having it "longer". I believe it can save random restart of the pod not really needed

Reply:
You're right, for a second I though I was changing only the agent timeout. I'll revert this

Reply:
Done in https://github.com/newrelic/nri-kubernetes/pull/346/commits/bbe936630551e15c66ff7efa6c1e049963b85645.
I kept the retries to 3 though, 4 with 10s mean waiting 40s which seems a bit excessive in my view